### ISSUE TITLE
Current / future behavior

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/smart-array-filter#10]
if you have

```
iupac: [{value:'triethylamine'},{value:'Et3N'}]
```

If would be nice to match this entry if you enter

```
iupac:triethyl
```

This means that when a parent match all the leaf is evaluated
What do you think ?

**Comment by targos** [comment]
I'm ok with that but then we need a way to enforce the current behavior

**Comment by lpatiny** [comment]
closed by d490b83bcfbd18d34d7f32322fb5e20b853c1950
